### DISCUSSION TITLE
I got a blocked host error

### DISCUSSION THREAD

**Discussion by 123ysys** [misskey-dev/misskey#13986] [General]
## 🥰 Expected Behavior

< !-- I should be able access the website when I type my misskey domain name
-->

## 📝 Steps to Reproduce

1. Install misskey application with installation bash script 
2. Type my domain 
3. See this error 

## 📌 Environment

Misskey version: The Latest, I guess 
Your OS: Ubuntu 18
Your browser: Chrome

**Comment by 123ysys** [comment]
![Screenshot_20221219-213925](https://user-images.githubusercontent.com/27964272/208441219-10fff16d-acb3-4e4a-aadd-eba43d7dbbf8.png)

It's just look like this.

**Comment by acid-chicken** [comment]
The error shown in the image looks like a typical Rails error screen, but [Misskey never uses Rails](https://github.com/search?q=org%3Amisskey-dev+rails) in any way.
Please provide more detailed information about your environment and how to reproduce this.

**Comment by 123ysys** [comment]
> The error shown in the image looks like a typical Rails error screen, but [Misskey never uses Rails](https://github.com/search?q=org%3Amisskey-dev+rails) in any way. Please provide more detailed information about your environment and how to reproduce this.

Hello. I searched it on the Internet and know it's like due to rails 6 but what confused me is I have no idea about where is the config.hosts.

However, I have installed rails before because there is a Mastodon service running on my VPS.

The more detailed steps to reproduce my error:
```
Originally I had mastodon installed on my machine and my misskey domain was using a subdomain of my mastodon domain.

1. Start install misskey using bash script

2. Choose the systemd

3. Choose the repository URL

4. Enter the name of the user

5. Enter my misskey's domain

6. Choose to install nginx on local

7 . Generate a Cloudflare Global API and copy and paste it from my Cloudflare account. However, it said that it failed to validate the API. Error message is: 
However, I ignored this error and move on.

8. I add the DNS records of subdomain of my mastodon domain as my misskey domain on my Cloudflare, but I didn't add the DNS records on my Domain Register.

9. Select no at the step of installing postgresql and enter the default values in sub option, like database name, password and port 

10. Select no at the step of installing redis, and Syslog, also enter the default values like previous step 

11. The installation processing finished, and I saw the Cloudflare validation I mentioned before.

12. Access my misskey's domain and see this error.
```

**Comment by acid-chicken** [comment]
Then perhaps, there is a Mastodon component still running; before starting the installation of Misskey, completely terminate the Mastodon server, disable it if it was serviced by systemd, etc., and delete it if necessary.
